astro/src/content/quickstarts/quickstart-php-laravel-api.mdx

```diff
@@ -0,0 +1,451 @@
+
+Your response should look like this:
+
+```json
+{"message":"We've called the police!"}
+```
+
+Nice, help is on the way!
+
+Now let's try as `customer@example.com` who has the role `customer`. Acquire a token for `customer@example.com`.
+
+<RemoteValue
+    url={frontmatter.coderoot + "/kickstart/kickstart.json"}
+    selector="$.variables"
+    codeLang="shell"
+    codeRenderer={(variables) => `
+curl --location 'http://localhost:9011/api/login' \\
+--header 'Authorization: ${variables.apiKey}' \\
+--header 'Content-Type: application/json' \\
+--data-raw '{
+  "loginId": "${variables.customerEmail}",
+  "password": "${variables.customerPassword}",
+  "applicationId": "${variables.applicationId}"
+}'`}/>
+
+Your response should look like this:
+
+```json
+{
+    "token": "eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCIsImtpZCI6InVOYl9iQzFySHZZTnZMc285VzRkOEprZkxLWSJ9.eyJhdWQiOiJlOWZkYjk4NS05MTczLTRlMDEtOWQ3My1hYzJkNjBkMWRjOGUiLCJleHAiOjE2ODkzNTQxMjMsImlhdCI6MTY4OTM1MzUyMywiaXNzIjoiaHR0cDovL2xvY2FsaG9zdDo5MDExIiwic3ViIjoiMDAwMDAwMDAtMDAwMC0wMDAwLTAwMDAtMjIyMjIyMjIyMjIyIiwianRpIjoiYjc2YWMwMGMtMDdmNi00NzkzLTgzMjgtODM4M2M3MGU4MWUzIiwiYXV0aGVudGljYXRpb25UeXBlIjoiUEFTU1dPUkQiLCJlbWFpbCI6ImN1c3RvbWVyQGV4YW1wbGUuY29tIiwiZW1haWxfdmVyaWZpZWQiOnRydWUsImFwcGxpY2F0aW9uSWQiOiJlOWZkYjk4NS05MTczLTRlMDEtOWQ3My1hYzJkNjBkMWRjOGUiLCJyb2xlcyI6WyJjdXN0b21lciJdLCJhdXRoX3RpbWUiOjE2ODkzNTM1MjMsInRpZCI6ImQ3ZDA5NTEzLWEzZjUtNDAxYy05Njg1LTM0YWI2YzU1MjQ1MyJ9.T1bELQ6a_ItOS0_YYpvqhIVknVMNeamcoC7BWnPjg2lgA9XpCmFA2mVnycoeuz-mSOHbp2cCoauP5opxehBR2lCn4Sz0If6PqgJgXKEpxh5pAxCPt91UyfjH8hGDqE3rDh7E4Hqn7mb-dFFwdfX7CMdKvC3dppMbXAGCZTl0LizApw5KIG9Wmt670339pSf5lzD38P9WAG5Wr7fAmVrIJPVu6yv2FoR-pMYD2lnAF63HWKknrWB-khmhr9ZKRLXKhP1UK-ThY1FSnmpp8eNblsBqCxf6WaYxYkdp5_F2e56M4sQwHzrg4P9tZGVCmMri4dShF3Ck7OGa7hel-iIPew",
+    "tokenExpirationInstant": 1689354123118,
+    "user": {
+        ...
+    }
+}
+```
+
+Now use that token to call `/api/make-change` with a query parameter `total=3.24`.
+
+```shell
+curl --cookie 'app.at=YOUR_TOKEN' --location 'http://localhost/api/make-change?total=3.24'
+```
+
+The response should look similar to below, however it has been formatted here for readability:
+
+```json
+{
+  "Message": "We can make change using 12 quarters 2 dimes 0 nickels 4 pennies",
+  "Change": [
+    {
+      "Denomination": "quarters",
+      "Count": 12
+    },
+    {
+      "Denomination": "dimes",
+      "Count": 2
+    },
+    {
+      "Denomination": "nickels",
+      "Count": 0
+    },
+    {
+      "Denomination": "pennies",
+      "Count": 4
+    }
+  ]
+}
+```
+
+So far so good. Now call the `/api/panic` endpoint with `-i` flag to see the headers of the response).
+
+```shell
+curl --cookie 'app.at=YOUR_TOKEN' -i --request POST 'http://localhost/api/panic'
+```
+
+Your should receive a `403 Forbidden` response with the following message and a huge stack trace (because you are in development mode):
+
```

Comment:
I did not get json.  I received html, with the 
![Screenshot 2023-10-26 at 7 56 17 AM](https://github.com/FusionAuth/fusionauth-site/assets/137117976/11bc7aff-60b0-46d9-9eee-632288f29cf7)

Reply:
I changed the doc to add an `Accept: application/json` header to force Laravel render JSON instead. Thanks